cli/command/run/cmd.go

```diff
@@ -531,6 +536,69 @@ func runTestcafeInCloud(p testcafe.Project, regio region.Region, tc testcomposer
 	return r.RunProject()
 }
 
+func runXcuitest(cmd *cobra.Command, tc testcomposer.Client, rs resto.Client, rc rdc.Client, as *appstore.AppStore) (int, error) {
+	p, err := xcuitest.FromFile(cfgFilePath)
+	if err != nil {
+		return 1, err
+	}
+	p.Sauce.Metadata.ExpandEnv()
+	applyDefaultValues(&p.Sauce)
+	overrideCliParameters(cmd, &p.Sauce)
+
+	// TODO - add dry-run mode
```

Comment:
When is that tackled? 🤔

Reply:
Espresso also has this TODO comment and I also decided to add it here.

Reply:
dry-run mode is for creating a test bundle but not uploading that. Do we need a dry-run mode for xcuitest and espresso? 🤔

Reply:
If the user wants to check the bundle before executing it then yes.

Reply:
The bundle being the app itself, i think it may not be making sense.
It could make sense when we would be supporting `.app` -> `.ipa` conversion.

Reply:
> If the user wants to check the bundle before executing it then yes.

Well, it's not just about the bundle but the configurations. This is especially useful as we multiple platform versions for emulators etc.. The benefit the user would therefore get is the ability to see what kind of jobs would have resulted from the given configuration. The archive/bundle, is just a small piece of this.

Reply:
I will create a separate ticket to add this feature.